compiler/rustc_smir/src/rustc_internal/mod.rs

```diff
@@ -43,6 +47,18 @@ impl<'tcx> Tables<'tcx> {
         self.def_ids.push(did);
         stable_mir::CrateItem(id)
     }
+
+    pub fn adt_def(&mut self, did: DefId) -> stable_mir::ty::AdtDef {
```

Comment:
I don't like that this is exactly the same as `item_def_id`. I can change this to make it DRY but unsure how much this is going to stay like this. @oli-obk ?

Reply:
Fixed!